lightning/src/sign/mod.rs

```diff
@@ -346,11 +359,14 @@ impl SpendableOutputDescriptor {
 	/// does not match the one we can spend.
 	///
 	/// We do not enforce that outputs meet the dust limit or that any output scripts are standard.
-	pub fn create_spendable_outputs_psbt(descriptors: &[&SpendableOutputDescriptor], outputs: Vec<TxOut>, change_destination_script: ScriptBuf, feerate_sat_per_1000_weight: u32, locktime: Option<LockTime>) -> Result<(PartiallySignedTransaction, u64), ()> {
+	pub fn create_spendable_outputs_psbt(descriptors: &[&SpendableOutputDescriptor], outputs: Vec<TxOut>, change_destination_script: ScriptBuf, feerate_sat_per_1000_weight: u32, locktime: Option<LockTime>, delayed_payment_basepoint: Option<&DelayedPaymentBasepoint>) -> Result<(PartiallySignedTransaction, u64), ()> {
```

Comment:
Updated the descriptor. The basepoint will also be stored in the channel monitor file, right? 
While we on the descriptor, would it be ok to add `get_claimable_balance` to it as well, so we have the view on the amount of funds this output unlocks vs what is the total amount of funds claimable? It would be useful for audit and accounting, but also for user reporting.